.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
   - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
 script:
   - mvn spotless:check -B -V
-  - mvn install -B -V -Pallow-snapshots,with-coverage,travis
+  - mvn install -B -V -Pallow-snapshots,with-coverage,travis -Dorg.slf4j.simpleLogger.log.org.apache.maven.plugins.shade=error
```

Comment:
do you still need this?

Reply:
Yes. Shading spams the log and travis then kills the build. We probably might want to do something about the tons of warnings generated by shade, but because I suppose we want to move to java modules as soon as possible, I'd keep it as it is.